src/Toolkit/Toolkit.iOS/UI/Controls/BookmarksView/BookmarksViewController.cs

```diff
@@ -0,0 +1,106 @@
+﻿// /*******************************************************************************
+//  * Copyright 2012-2018 Esri
+//  *
+//  *  Licensed under the Apache License, Version 2.0 (the "License");
+//  *  you may not use this file except in compliance with the License.
+//  *  You may obtain a copy of the License at
+//  *
+//  *  http://www.apache.org/licenses/LICENSE-2.0
+//  *
+//  *   Unless required by applicable law or agreed to in writing, software
+//  *   distributed under the License is distributed on an "AS IS" BASIS,
+//  *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  *   See the License for the specific language governing permissions and
+//  *   limitations under the License.
+//  ******************************************************************************/
+
+using System;
+using UIKit;
+
+namespace Esri.ArcGISRuntime.Toolkit.UI.Controls
+{
+    /// <summary>
+    /// View controller to simplify the presentation of a modal bookmarks view.
+    /// Present this view controller modally to show a list of bookmarks to choose from modally.
+    /// Show <see cref="BookmarksView" /> directly in your view if you want to
+    /// lay out and manage the display of the bookmarks list yourself.
+    /// </summary>
+    public class BookmarksViewController : UIViewController
```

Comment:
Do we want to start providing modal controllers for all our controls, or should this be up to the app developer to do?